server/utils/pagePropsUtil.js

```diff
@@ -0,0 +1,10 @@
+export const userProps = (req) => {
```

Comment:
`userProps` feels incorrect to me. This isn't a prop or multiple props, it is a user object that is constructed from the request object.

Reply:
[78b1245](https://github.com/AMP-SCZ/dpdash/pull/84/commits/78b12458361f85aeb0919c885be4ca8c8f6c047b)